lib/Driver/ToolChains.cpp

```diff
@@ -585,8 +585,8 @@ ToolChain::constructInvocation(const CompileJobAction &job,
   if (context.OI.CompilerMode == OutputInfo::Mode::SingleCompile) {
     context.Args.AddLastArg(Arguments, options::OPT_emit_symbol_graph);
     context.Args.AddLastArg(Arguments, options::OPT_emit_symbol_graph_dir);
-    context.Args.AddLastArg(Arguments, options::OPT_include_spi_symbols);
   }
+  context.Args.AddLastArg(Arguments, options::OPT_include_spi_symbols);
```

Comment:
I think this will need a matching change in swift-driver in CompileJob.swift.

https://github.com/apple/swift-driver/blob/main/Sources/SwiftDriver/Jobs/CompileJob.swift#L351-L355